.azure_pipelines/ci-pipeline-makefile.yml

```diff
@@ -39,7 +39,8 @@ jobs:
           sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
           sudo apt-get update
           while sudo lsof /var/lib/dpkg/lock-frontend | grep dpkg; do sleep 3; done
-          sudo apt-get install build-essential python3-setuptools libmbedtls-dev docker-ce -y 
+          sudo apt-get install build-essential python3-setuptools libmbedtls-dev docker-ce lldb-8 -y
+          sudo ln -s /usr/lib/llvm-8/bin/lldb /usr/bin/lldb
```

Comment:
The /usr/bin/lldb symlink is not needed. oelldb figures out the right lldb executable:
https://github.com/openenclave/openenclave/blob/ee0032655e5e8b6f847cd2a3a0008d1464254e3c/debugger/oelldb#L25-L32